server/database/queries/addUser.js

```diff
@@ -1,12 +1,12 @@
 const dbConnection = require('../dbConnection')
 const addUser = (name, email, password, next) => {
   return dbConnection
-    .query('INSERT INTO users (name,email,password) values ($1,$2,$3)', [
+    .query('INSERT INTO users (name, email, password) values ($1, $2, $3)', [
       name,
       email,
       password
     ])
-    .then(res => res.rows)
+    .then(res => 'inserted')
```

Comment:
it is usually useful to return the user object minus password. 
You can return the name to use it.
No changes required here. It's just FYI.